### PR TITLE
More endpoints

### PR DESCRIPTION
**Pull request by jcrugzz**
Add a couple endpoints, webhooks being most important.Works with a proper token locally.

### CODE REVIEW

File: endpoints/collaborators.js [hunk 1]
```diff
@@ -37,6 +37,31 @@ Collaborators.prototype.get = function get(args) {
...
+/**
+ * Add a collaborator to the owner/repo given
+ *
+ * @param {String} project
+ * @param {Object}
+ *
+ */
+Collaborators.prototype.add = function add(args) {
+  args = this.api.args(arguments);
+  args.options = this.options(args.options);
```
Comment (Swaagie) [inline endpoints/collaborators.js]: wouldn't this bleed out to the `args.options` object outside this method?

File: endpoints/collaborators.js [hunk 2]
```diff
@@ -37,6 +37,31 @@ Collaborators.prototype.get = function get(args) {
   );
 };
 
+/**
+ * Add a collaborator to the owner/repo given
+ *
+ * @param {String} project
+ * @param {Object}
+ *
```
Comment (3rd-Eden) [inline endpoints/collaborators.js]: `@api public`

File: endpoints/collaborators.js [hunk 3]
```diff
@@ -37,6 +37,31 @@ Collaborators.prototype.get = function get(args) {
...
+  if (!user) { return arg.fn(new Error('user is a required options')); }
+
+  var project = this.api(args.str);
+
+  return this.send(
+    ['repos', project.user, project.repo, 'collaborators', user],
+    args.options,
+    args.fn
+  );
+
```
Comment (3rd-Eden) [inline endpoints/collaborators.js]: Syntax nazi - whitespace.

File: endpoints/collaborators.js [hunk 4]
```diff
@@ -37,6 +37,31 @@ Collaborators.prototype.get = function get(args) {
...
+ * @param {Object}
+ *
+ */
+Collaborators.prototype.add = function add(args) {
+  args = this.api.args(arguments);
+  args.options = this.options(args.options);
+
+  var user = args.options.username || args.options.user;
+
+  if (!user) { return arg.fn(new Error('user is a required options')); }
```
Comment (3rd-Eden) [inline endpoints/collaborators.js]: arg is undefined, should be args.

**Comment by 3rd-Eden** [comment]
You have been syntax and JSDoc fixed. Just waiting for travis-ci to stop complaining and then i'll pull it in and release a new minor.

**Pull request merged**